src/database/migrations/20220920054800-create-launches.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("launches", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      site_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      primary_domain_source: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      primary_domain_target: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
```

Comment:
correct me if i'm wrong but do source/target here refer respectively to the `<some_random_letters>.amplify.com` and `<agency_domain>.com` respectively?

Reply:
for clarity:
source: `<agency_domain>.com`
target: `<some_random_letters>.amplify.com`

Reply:
thanks for clarifying!